.gitignore

```diff
@@ -2,3 +2,5 @@
 .*sw[op]
 *.py[co]
 *.egg-info
+persistent
+runtime
```

Comment:
Please prefix any rules possible with a `/` prefix.
That will require they match in the top directory (relative to the gitgnore file).

It's a good way to make sure you don't accidentally match something else later.